rules/import_middleman.bzl

```diff
@@ -239,11 +249,17 @@ def _file_collector_rule_impl(ctx):
             dynamic_framework_dirs.append(f)
     objc_provider_fields["dynamic_framework_file"] = depset(dynamic_framework_file)
 
-    replaced_frameworks = replaced_dyanmic_framework.values() + replaced_static_framework.replaced.values()
+    all_replaced_frameworks = replaced_dyanmic_framework.values() + replaced_static_framework.replaced.values()
+
+    # This slightly varies in Bazel 6.x.x
```

Comment:
Can you expand this comment to explain what is different between the versions and needs unique handling for each?

Reply:
@jerrymarino correct me if I'm wrong but I believe what this is doing is handling only dynamic fmws in Bazel 6 since static fmws are handled here: https://github.com/bazel-ios/rules_ios/pull/766. If that's the case a small comment with this context would be preferred IMO.

Reply:
Yeah you got it @thiagohmcruz - the `Notes` blurb now links this:  https://github.com/bazel-ios/rules_ios/pull/767/files#diff-e213489d3a65f173ffa46728826b06f786a5d63666e3fe7101493397ee12f948R2